scripts/PyChop/Instruments.py

```diff
@@ -0,0 +1,903 @@
+"""
+This module is a wrapper around a set of instrument parameters (to be read from a YAML file)
+and methods which then call either Chop.py or MulpyRep.py to do the resolution calculations.
+"""
+
+from __future__ import (absolute_import, division, print_function)
+from six import string_types
+import numpy as np
+import yaml
+import warnings
+import copy
+try:
+    from . import Chop, MulpyRep
+except:
```

Comment:
Sorry, this was in because I was testing code within the PyChop directory and importing `Instruments` directly so the relative import here fails. Anyway, this is no longer needed, so I've removed the `try` block and just left the relative imports.